test/vinyl-luatest/gh_6568_test.lua

```diff
@@ -0,0 +1,82 @@
+local cluster = require('test.luatest_helpers.cluster')
```

Comment:
1. We usually name tests so that it's clear what it does, e.g. `gh_5568_read_only_reason_test.lua`. Please rename.
2. The test takes too long to complete (~30 seconds). Please try to reduce the test run time down to 3-5 seconds max.

Reply:
1. Thought it would be be better for the sake of brevity to keep the details out of the way (IMO, people still go look up the corresponding issue). Fixed: provided a brief description in the test name.